### ISSUE TITLE
[BUG] Popup Height & Long Text

### ISSUE THREAD

**Issue by DENLN** [CommunityToolkit/Maui#1353]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Did you read the "Reporting a bug" section on Contributing file?

- [X] I have read the "Reporting a bug" section on Contributing file: https://github.com/CommunityToolkit/Maui/blob/main/CONTRIBUTING.md#reporting-a-bug

### Current Behavior

Popup height not correctly calculated for long text

![IMG_20230819_211500](https://github.com/CommunityToolkit/Maui/assets/4215311/db58ccb7-4978-4efa-a8e8-b6052cc1162b)

### Expected Behavior

I expected that all text would be visible and the height would adjust automatically

### Steps To Reproduce

1. Open and run soluition from reproduction repository
2. Click Button "Show Popup"

### Link to public reproduction project repository

https://github.com/DENLN/PopupLabelLong

### Environment

```markdown
- .NET MAUI CommunityToolkit: 5.3
- OS: Android 13
- .NET MAUI: 7.0.92
```

**Comment by cat0363** [comment]
The Popup's content size is not calculated correctly when the Popup's Size property is not specified.
In the PR I am currently creating, I got the following execution results.

<img width="300" src="https://github.com/CommunityToolkit/Maui/assets/125236133/4fd1de99-e9d4-4972-acba-c5d29d4d2d13" />

Along with this fix, I would like to fix it because it is not the intended behavior when Fill is specified for VerticalOptions and HorizontalOptions of Popup.

[When HorizontalOptions is Fill]
I changed the VerticalOptions to:

<table>
  <tr>
    <td>[Start]</td>
    <td>[Center]</td>
    <td>[End]</td>
  </tr>
  <tr>
    <td><img width="250" src="https://github.com/CommunityToolkit/Maui/assets/125236133/bdce3feb-1061-46e0-8c58-8c1eed0bf7a7" /></td>
    <td><img width="250" src="https://github.com/CommunityToolkit/Maui/assets/125236133/9446584d-fdba-4627-bfc3-8b51e643390a" /></td>
    <td><img width="250" src="https://github.com/CommunityToolkit/Maui/assets/125236133/ff39cdcd-5753-418c-9a40-8fd1c094b171" /></td>
  </tr>
</table>

[When VerticalOptions is Fill]
I changed the HorizontalOptions to:

<table>
  <tr>
    <td>[Start]</td>
    <td>[Center]</td>
    <td>[End]</td>
  </tr>
  <tr>
    <td><img width="250" src="https://github.com/CommunityToolkit/Maui/assets/125236133/c435c083-78bc-4f55-a953-b21984529b10" /></td>
    <td><img width="250" src="https://github.com/CommunityToolkit/Maui/assets/125236133/c7c469ab-5260-48d7-ad36-fa0a485e166b" /></td>
    <td><img width="250" src="https://github.com/CommunityToolkit/Maui/assets/125236133/5dffdf39-f012-416c-9895-6f711eb63aa0" /></td>
  </tr>
</table>

[When both HorizontalOptions and VerticalOptions are Fill]

<img width="250" src="https://github.com/CommunityToolkit/Maui/assets/125236133/42d763e0-381a-4312-96b6-efe8f9a99445" />

I will make a PR tomorrow.

**Comment by r2d2Proton** [comment]
Popup shows less on my Windows 11 desktop than on my Android phone. The desktop is truncated.

**Comment by cat0363** [comment]
@r2d2Proton , A similar issue on Windows will be fixed in PR #1350.

**Comment by VladislavAntonyuk** [comment]
Duplicates #1150
